docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '2.1'
+services:
+  nginx:
+    image: nginx
+    ports:
+      - '4000:4000'
+    volumes:
+      - assets:/usr/share/nginx/html:ro
+      - ./nginx.conf:/etc/nginx/conf.d/default.conf
+  site:
+    build: .
+    volumes:
+      - .:/opt/certbot/
```

Comment:
This volume caused some issues for me. `npm install` / `_docs.sh depend/install` / `gulp build`  are run in the container as part of the Dockerfile, so my host directory did not initially have node_modules / build products.